src/Orchard.Web/Modules/Orchard.DynamicForms/Elements/PlaceholderableFormElement.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Orchard.Layouts.Helpers;
+
+namespace Orchard.DynamicForms.Elements {
+    public abstract class PlaceholderableFormElement : LabeledFormElement {
```

Comment:
Although it's creative, I'm not a fan of the word "placeholderable" :)
Also, why not add `Placeholder` to the `LabeledFormElement` directly, instead of subclassing?

PS. Great to see you back! :)